kube-client/src/client/config_ext.rs

```diff
@@ -23,6 +23,9 @@ pub trait ConfigExt: private::Sealed {
     /// Optional layer to set up `Authorization` header depending on the config.
     fn auth_layer(&self) -> Result<Option<AuthLayer>>;
 
+    /// Use certificate data from exec
+    fn exec_identity_pem(&self) -> Option<Vec<u8>>;
```

Comment:
I don't think we should make this method public. You can add `impl Config` in this file with the method.